src/main/java/pl/javalon4/finalproject/dto/LinkDto.java

```diff
@@ -0,0 +1,18 @@
+package pl.javalon4.finalproject.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import pl.javalon4.finalproject.enity.AppUser;
+import pl.javalon4.finalproject.enity.LinkCategory;
+import pl.javalon4.finalproject.enity.LinkStatus;
+
+@Data
+@AllArgsConstructor
+public class LinkDto {
+
+    private String url;
+    private String description;
+    private LinkStatus linkStatus;
```

Comment:
LinkStatus and LinkCategory should be replaced by equivalent types from dto layer.